scripts/20210616_remove-permissionList-read/remove-permissionList-read.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable */
+
+// Number of forms with permissionList.read key - expect 0 after running update
+db.forms.count({
+  permissionList: { $elemMatch: { read: { $exists: true } } }
+})
+
+{
+  // Number of objects in permissionList with read key - expect 0 after running update
+  const permissionListWithReadKey = [
+    {
+      $match: {
+        permissionList: { $elemMatch: { read: { $exists: true } } }
+      },
+    },
+    { $project: { permissionList: 1 } },
+    { $unwind: '$permissionList' },
+    { $match: { 'permissionList.read': { $exists: true } } },
+    { $count: 'numObjs' }
+  ]
+
+  db.getCollection('forms').aggregate(permissionListWithReadKey)
+}
+
+// Update - Remove read key
+db.forms.update(
+  { 'permissionList.read': { $exists: true } },
+  { $unset: { 'permissionList.$[elem].read': 1 } },
+  { arrayFilters: [{ 'elem.read': { $exists: true } }], multi: true }
```

Comment:
again no need to change but this is equivalent to this:
```suggestion
  { $unset: { 'permissionList.$[].read': 1 } },
  { multi: true }
```